src/app/components/loading-spinner.tsx

```diff
@@ -9,6 +12,13 @@ export function LoadingSpinner(props: FlexProps) {
 }
 
 export function FullPageLoadingSpinner(props: FlexProps) {
+  const [hasCreatedAccount, setHasCreatedAccount] = useHasCreatedAccount();
+
+  useEffect(() => {
+    hasCreatedAccount && setHasCreatedAccount(false);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
We should try and keep this component dumb. Rather than adding implementation details of `useHasCreatedAccount` to `FullPageLoadingSpinner`, it would be better to make a new component entirely

Reply:
Agree, was trying to think of a better way to do this. `FullPageLoadingSpinner` is only used in `app.tsx` atm but might be shared elsewhere later. I'll move it from here.